hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -36,7 +36,7 @@ objects:
     spec:
       clusterDeploymentSelector:
         matchLabels:
```

Comment:
doesn't `matchLabels` require a value? and empty map maybe?

https://github.com/openshift/hive/blob/688681d11095a56c1cae8fc5a01e9bff92455462/apis/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1101